utils/create_dummy_models.py

```diff
@@ -999,6 +1084,37 @@ def build(config_class, models_to_create, output_dir):
     return result
 
 
+def build_tiny_model_summary(results):
+    """Build a summary: a dictionary of the form
+    {
+      model architecture name:
+        {
+          "tokenizer_classes": [...],
+          "processor_classes": [...]
+        }
+      ..
+    }
+    """
+    tiny_model_summary = {}
```

Comment:
Main change 3: this method will produce the entries we might want to add `tests/utils/tiny_model_summary.json` (as long as we still want to keep this file for now)